percona-toolkit 3.1.0

bayandin opened the pull request:
Created with `brew bump-formula-pr`.

Moisan commented:
It seems like the `high_sierra` bottle didn't get build. @BrewTestBot test this please.

bayandin commented:
Problems with bottles again: 
```
16:38:40 Error: percona-toolkit-3.1.0.catalina.bottle.tar.gz is already published. Please remove it manually from
```

I've removed the bottles from bintray and fixed the url (the previous link to source was related to Debian somehow)
